honeybot/plugins/password_generator.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+[password_generator.py]
+Password Generator Plugin
+
+[Author]
+Nishant, JPMorgan Chase & Co.
+
+[About]
+sends different type of passwords
+
+[Commands]
+>>> .passgen <<length>>
+returns alphabetic password of specified length
+
+>>> .passgensecure <<length>>
+returns secure alphanumeric password of specified length
+
+>>> .passgenalphanum <<length>>
+returns alphanumeric password of specified length
+
```

Comment:
excellent, had in mind to gen different pswd but forgot to put in descrip. maybe a nice addition might be: secure easy to remember password. like i-lov3@swimming4lif3

Reply:
secure easy to remember password?

Reply:
like it generates words but modifies to something similar to i-lov3@swimming4lif3 such that it contains alphanum + symbols. find replacement for i etc in symbols like we replaced e by 3

Reply:
opening an issue for that